.github/workflows/inductor-perf-test-nightly.yml

```diff
@@ -33,13 +63,40 @@ jobs:
           { config: "inductor_torchbench_perf", shard: 3, num_shards: 3, runner: "linux.gcp.a100.large" },
         ]}
 
-  linux-bionic-cuda11_8-py3_10-gcc7-inductor-test:
+  linux-bionic-cuda11_8-py3_10-gcc7-inductor-test-5days:
     name: cuda11.8-py3.10-gcc7-sm80
     uses: ./.github/workflows/_linux-test.yml
     needs: linux-bionic-cuda11_8-py3_10-gcc7-inductor-build
+    if: github.event.schedule == '0 1,2,4,5,6 * * *'
     with:
-      build-environment: linux-bionic-cuda11.8-py3.10-gcc7-sm80
+      # training only
+      build-environment: linux-bionic-cuda11.8-py3.10-gcc7-sm80-training-true-inference-false-default-true-dynamic-true-cudagraphs-true
```

Comment:
This is wicked, the way you pass the parameters into test.sh via `build-environment`